### ISSUE TITLE
Server - Kirin Tor // Wintergrasp not starting.

### ISSUE THREAD

**Issue by BlanketBearer** [dalaranwow/dalaran-wow#4792]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

There is no countdown timer for Wintergrasp to start

**Expected behaviour**: Tell us what should happen instead

A countdown timer should be present at the bottom of the screen for the start of Wintergrasp.

**Steps to reproduce the problem**:

1.  Go to Wintergrasp in Kirin Tor.
2. Notice that there is no countdown to the start of Wintergrasp at the bottom of the screen. 
3. 

**Include proofs for this behaviour**

http://wowwiki.wikia.com/wiki/Wintergrasp

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Wintergrasp Zone ID: 4197

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by ashido96** [comment]
The only reason the kirin tor server exists is to allow people to post on the forums because of janky website set up, play on algalon.

**Comment by BlanketBearer** [comment]
Except, I don't want to play on Algalon, and neither do any of the people in my guild. I want to play on Kirin Tor. The original reason I wanted to start on this realm is for the lack of PVP, not a high population. There are only two issues with this realm that we are having issues with that are prohibitive for advancement at level 80. I'm not asking for every little bug to get fixed, just the major ones.

**Comment by Shaket** [comment]
@BlanketBearer I totally agree, if there are two realms I understand my right to play undistinctly on either of them. If there was only one I would play on it, or I wouldn't play on this server at all. Said this, as players is our right to ask for some fixes just like Algalon players' demmand their own. 
Thanks!

**Comment by Huppelmus** [comment]
Thank you for creating this topic, Blanket. I wholeheartedly agree with your sentiment, and hope these issues will be fixed. 
I understand that Dalaran-WoW wants to focus on their progressive PvP realm; they put a lot of effort into creating a progressive realm, after all. But I believe their ushering of players to their other realm was a mistake. Free-for-all PvP isn't interesting for everyone, and neither is heavily competitive play. 

It is my hope that, once Dalaran-WoW gets done with its progression, they'll shift their focus back to Kirin Tor.

Until then, the lack of customer support for Kirin Tor is really off-putting, and it's not inviting any of us to spend money on donations... And that is, after all, what the servers run on.

**Comment by Karkazz** [comment]
Wintergrasp used to work on Kirin-Tor back in 2013, I assume they disabled it because of the low population.

**Comment by BlanketBearer** [comment]
I don't see why they would disable it still. It seems odd to not include every fix from the other realm to this one. There are a lot of things that work on Algalon, that just do not function on Kirin Tor. These problems would be trivial to fix, if they copied the core, disabled PvP, and remove progress gates.

**Comment by Karkazz** [comment]
Are you playing on horde?
I've whispered some horde people with the suggestion to queue BG on set date/times but I've always seemed to hit a brick wall.
Answers like "go play to Algalon if you want PVP, /ignore", they don't seem to grasp the simple fact that there's PVE played on PVP servers and vice versa.
I'm on alliance btw, could arrange that like 7 to 15 alliances queue BG on set times.

**Comment by Rushor** [comment]
Please discuss this topic in the forum and open a new forumthread for it. Thanks.
